src/griptape_nodes/retained_mode/managers/library_manager.py

```
@@ -951,7 +951,10 @@ async def register_library_from_file_request(self, request: RegisterLibraryFromF
             # Assign them into the config space.
             for library_data_setting in library_data.settings:
                 # Does the category exist?
-                get_category_request = GetConfigCategoryRequest(category=library_data_setting.category)
+                get_category_request = GetConfigCategoryRequest(
+                    category=library_data_setting.category,
+                    failure_log_level=logging.DEBUG,  # Missing category is expected, suppress error toast
```

I can see how this is overriding the default log level for the engine logs by moving the logging logic from `ResultDetails.__init__` to `EventManager._handle_request_core`, but how does it suppress the error toast?

I'm assuing you mean the error toast "notification" widget on the frontend. Is there another related change on the frontend, or does the frontend magically create toasts for error logs (so fixing the engine logs would implicitly fix the frontend error toasts)?